nautobot/extras/models/jobs.py

```diff
@@ -294,20 +298,29 @@ def job_task(self):
 
     @property
     def task_queues(self):
-        """Deprecated backward-compatibility property for the list of Celery queues for this Job."""
-        return self.job_queues.filter(queue_type=JobQueueTypeChoices.TYPE_CELERY).values_list("name", flat=True)
+        """Deprecated backward-compatibility property for the list of queue names for this Job."""
+        return self.job_queues.values_list("name", flat=True)
 
     @task_queues.setter
     def task_queues(self, value):
+        job_queues = []
         # value is going to be a comma separated list of queue names
         if isinstance(value, str):
             value = value.split(",")
         for queue in value:
             try:
-                job_queue = JobQueue.objects.get(name=queue, queue_type=JobQueueTypeChoices.TYPE_CELERY)
+                job_queues.append(JobQueue.objects.get(name=queue))
             except JobQueue.DoesNotExist:
-                raise ValidationError(f"Celery Job Queue {queue} does not exist in the database.")
-            JobQueueAssignment.objects.get_or_create(job_queue=job_queue, job=self)
+                raise ValidationError(f"Job Queue {queue} does not exist in the database.")
+        self.job_queues.set(job_queues)
+
+    @property
+    def task_queues_override(self):
+        return self.job_queues_override
+
+    @task_queues_override.setter
+    def task_queues_override(self, value):
```

Comment:
Do we need a data validation check here? To make sure that value is boolean and such? Or maybe that's already handled by the booleanfield itself.

Reply:
In general our ORM API doesn't do these kinds of explicit type checks, though that's not to say it *shouldn't*. Can add it if you think it's a significant risk.